### ISSUE TITLE
[Linux/Vulkan] Client Freezes with Accept Notification

### ISSUE THREAD

**Issue by cluoma** [ValveSoftware/Dota-2#1391]
The game client freezes when the matchmaking Accept notification pops. This happens even during the first notification after restarting Steam and Dota 2.

Maybe related to [#1387](https://github.com/ValveSoftware/Dota-2/issues/1387)?

System info: https://gist.github.com/cluoma/ffbb413b97d961f2807a2531cd0bb821
Client log: https://gist.github.com/cluoma/8fef929bbcb85b83dca8cd7f6b6f9dff

The last line of the client log:
`[RenderSystem] Attempting to purge m_fencedResourceList.Count() = 177, m_pendingFencedResourceQueue.Count() = 2, nCurrentCmdBuffersAvailable = 0, nAllocatedCommandBuffers = 1500`

was repeated many, many times and managed to fill a 1.2GB text file in the short time, 30 seconds to a minute, before I killed the process.

**Comment by gdrewb-valve** [comment]
These particular messages are specific to Vulkan.  Do things work if you use OpenGL?

**Comment by cluoma** [comment]
Yup, switching to openGL seems to fix the issue. Thanks.

**Comment by Plagman** [comment]
Are you runnning Dota in fullscreen with unredirection enabled? Does the issue also happen in Vulkan + windowed mode?

**Comment by cluoma** [comment]
Yes I usually run fullscreen. I tested, with vulkan enabled, and did not encounter freezing in windowed mode.

As far as I know unredirection is no longer supported in KWin as of 5.8. I run 5.8.8. (https://blog.martin-graesslin.com/blog/2016/08/opengl-changes-in-kwin-compositing/)

**Comment by dmarcuse** [comment]
Any update on this? I'm still having this issue, with i3wm and vulkan. [System info](https://pastebin.com/u7QFJf0P).
